tests/integration-tests/tests/performance_tests/test_starccm.py

```diff
@@ -0,0 +1,95 @@
+import json
+import logging
+
+import boto3
+import pytest
+from assertpy import assert_that
+
+from remote_command_executor import RemoteCommandExecutionError, RemoteCommandExecutor
+
+# timeout in seconds
+STARCCM_INSTALLATION_TIMEOUT = 1800
+STARCCM_JOB_TIMEOUT = 600
+STARCCM_LICENCE_SECRET = "perf_integ_secret"
```

Comment:
the name of the secret should be strictly related to StarCCM. `perf_integ_secret` is not enough to clarify that the secret is related to StarCCM. What about `starccm-license-secret`?

Reply:
Also, let's remember that this secret must be deployed in the accounts where this test is going to be executed, not only in personal acco8unts.